### ISSUE TITLE
Documentation asks for S3StaticStorage, but S3StaticStorage is not even mentioned anywhere in the whole repo.

### ISSUE THREAD

**Issue by aranmohyeddin** [jschneier/django-storages#971]
You have this [Here](https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html#settings):
```
STATICFILES_STORAGE = 'storages.backends.s3boto3.S3StaticStorage'
```
but S3StaticStorage is not even mentioned anywhere in the whole [repo](https://github.com/jschneier/django-storages/search?q=S3StaticStorage).

**Comment by jschneier** [comment]
https://github.com/jschneier/django-storages/blob/8ddc179e562ea4bf3a61f73b4c5fd17ea7edbd58/storages/backends/s3boto3.py#L588

**Comment by jschneier** [comment]
My bad that the docs seems to go up before I release.

**Comment by edumats** [comment]
Same problem here, my version is 1.11 and still can't upgrade it. The S3StaticStorage does not exist and was giving me an error when I was following the documentation

**Comment by jschneier** [comment]
I've just released 1.11.1, that will cover what you need.
